### PR TITLE
Feat/update colors 3

### PR DESCRIPTION
**Pull request by iqmulator**

### CODE REVIEW

File: src/nodejs/styles/colors.css [hunk 1]
```diff
@@ -363,54 +433,55 @@
...
     --skeleton: var(--gray-05);
     --skeleton-border: var(--gray-10);
-    --scrollbar: var(--gray-30);
+    --scrollbar: var(--ash-40);
     --quote: var(--gray-10);
     --place: var(--indigo-80);
     --menuitem-hover: var(--white-05);
     --menuitem-hover-warning: var(--magenta-50-10);
-    --bubble: var(--indigo-70);
+    --bubble: var(--indigo-60);
```
Comment (alexis-kochetov) [inline src/nodejs/styles/colors.css:484]: --indigo-70

File: src/dotnet/Chat.UI.Blazor/Components/ChatList/ChatListSortButton.razor [hunk 2]
```diff
@@ -7,7 +7,7 @@
 }
 
 <ButtonRound
-    Class="chat-list-sort-btn btn-sm"
+    Class="chat-list-sort-btn btn-sm transparent"
```
Comment (crui3er) [inline src/dotnet/Chat.UI.Blazor/Components/ChatList/ChatListSortButton.razor:10]: I think better add 'transparent' to 'chat-list-sort-btn' in css

File: src/dotnet/Chat.UI.Blazor/Components/ChatMessageEditor/chat-message-editor.css [hunk 3]
```diff
@@ -75,13 +75,13 @@
     @apply flex-none flex items-center justify-center;
     @apply h-12 w-10;
     @apply bg-transparent;
-    @apply text-03;
+    @apply text-round-title;
```
Comment (crui3er) [inline src/dotnet/Chat.UI.Blazor/Components/ChatMessageEditor/chat-message-editor.css:78]: weird name for text color 'round-title'

File: src/dotnet/Chat.UI.Blazor/Components/MarkupEditor/markup-editor.css [hunk 4]
```diff
@@ -31,7 +31,8 @@
 /* Placeholder */
 .markup-editor .editor-content:empty:not(:focus):before {
     content: attr(data-placeholder);
-    @apply text-counter md:text-03 !opacity-100 truncate;
+    /*@apply text-counter md:text-03 !opacity-100 truncate;*/
```
Comment (crui3er) [inline src/dotnet/Chat.UI.Blazor/Components/MarkupEditor/markup-editor.css]: remove code in comments

File: src/dotnet/Chat.UI.Blazor/Components/RecordingSubHeader/RecordingSubHeader.razor [hunk 5]
```diff
@@ -7,26 +7,31 @@
...
+        @if (isWarning) {
+            <i class="icon-alert-triangle text-banner-warning"></i>
+        } else {
+            <div class="c-round"></div>
+        }
     </div>
 
-    <div class="c-line-1">
-        @if (m.RecordingChat != null) {
+    @if (m.RecordingChat != null) {
```
Comment (crui3er) [inline src/dotnet/Chat.UI.Blazor/Components/RecordingSubHeader/RecordingSubHeader.razor]: Did you check how it looks like when RecordingChat != null and StopRecordingAt != null ?

**Review by crui3er — CHANGES_REQUESTED** [review]

**Pull request merged**